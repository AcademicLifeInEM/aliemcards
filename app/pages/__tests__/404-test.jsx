import React from 'react';
import renderer from 'react-test-renderer';
import FourOhFour from '../404';

describe('<404 />', () => {
  it('should render correctly', () => {
    const component = renderer.create(
      <FourOhFour />
    );
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
