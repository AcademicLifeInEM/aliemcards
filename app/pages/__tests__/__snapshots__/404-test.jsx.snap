exports[`<404 /> should render correctly 1`] = `
<div>
  <h3>
    404 page not found
  </h3>
  <p>
    We are sorry but the page you are looking for does not exist.
  </p>
</div>
`;
